wpilibc/athena/include/SpeedControllerGroup.h

```diff
@@ -0,0 +1,38 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016-2017. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <vector>
+
+#include "SpeedController.h"
+
+namespace frc {
+
+class SpeedControllerGroup : public SpeedController {
+ public:
+  SpeedControllerGroup() = default;
+  SpeedControllerGroup(
```

Comment:
SumNode.h and SumNode.inc from the CtrlSys patch (https://github.com/wpilibsuite/allwpilib/pull/544/files) provide an example of variadic templates. That should help add a variadic constructor here in addition to the initializer list one. Same thing could be applied to Add() to replicate the Java version.